### PR TITLE
FIX SSR

### PR DESCRIPTION
**Pull request by KevinGruber**
fixssr Move 'removeListener' and 'addlistener' from util.js to Lightbox.js;
add checks for 'window'

### CODE REVIEW

**Comment by pstadler** [comment]
Thanks for your pull request. Could you please make sure to keep the indentation?

**Comment by KevinGruber** [comment]
Yes of course.
Sry we just used my github account I will direct it tomorrow to the developer how made the Changes. What are your indent settings? Looks like he accidentally reformatted with our settings.

**Comment by pstadler** [comment]
two spaces I guess. Maybe we should add an `.editorconfig`.

**Comment by KevinGruber** [comment]
Would help with that :-)
Is ok👌

File: src/LightBox.js [hunk 1]
```diff
@@ -50,11 +50,17 @@ export default class LightBox extends Component {
   }
 
   componentDidMount() {
-    addListener('message', this.onMessage)
+    if (typeof window !== 'undefined') {
```
Comment (pstadler) [inline src/LightBox.js]: The lifecycle method `componentDidMount` is not called when rendered on the server, you can safely remove this check.

File: src/LightBox.js [hunk 2]
```diff
@@ -50,11 +50,17 @@ export default class LightBox extends Component {
...
+        const addListener = window.addEventListener || window.attachEvent
+        addListener('message', this.onMessage)
+    }
   }
 
   componentWillUnmount() {
-    removeListener('message', this.onMessage)
+    if (typeof window !== 'undefined') {
```
Comment (pstadler) [inline src/LightBox.js]: The lifecycle method `componentWillUnmount` is not called when rendered on the server, you can safely remove this check.

**Comment by KevinGruber** [comment]
Done :-)

**Review by pstadler — CHANGES_REQUESTED** [review]

File: src/LightBox.js (hunk 1 above)
Comment (KevinGruber) [reply src/LightBox.js]: Yes that is correct.
I didn't review it before.
I will forward it...

File: src/LightBox.js (hunk 2 above)
Comment (KevinGruber) [reply src/LightBox.js]: Yes that is correct
I will forward it...

**Comment by KevinGruber** [comment]
check for window removed.

**Comment by pstadler** [comment]
Thanks for your cooperation 🍻

**Review by pstadler — APPROVED** [review]

**Comment by pstadler** [comment]
This has been released with [1.0.5](https://github.com/datatrans/react-datatrans-light-box/releases/tag/1.0.5)

**Pull request merged**